compact/hcal/backward_LFHCAL_geo.xml

```diff
@@ -0,0 +1,1357 @@
+
+      The HCal forward insert shape is cut out from this detector
+
+      Currently don't have an implementation of electronics, so just leave that area (final 10 cm) empty
+      So then, actual detector area is "nHCal_LFHCAL_length - LFHCALElectronicsThickness" long
+    </documentation>
+    <detector
+      id="HcalEndcapN_ID"
+      name="HcalEndcapN"
+      type="epic_LFHCAL"
+      readout="HcalEndcapNHits"
+      vis="InvisibleWithDaughters"
+      calorimeterType="HAD_ENDCAP"
+      renderComponents="1"
+      allSensitive="0"
+      >
+
+      <position x="0" y="0" z="-HcalEndcapN_zmin-nHCal_LFHCAL_length+23.94"/>
+      <dimensions
+        z="nHCal_LFHCAL_length"
+        rmin="20*cm"
+        rmax="FluxEndcapN_exterior_rmax"
+        x="3*EightM_OuterWidth"
+        y="6*EightM_OuterHeight"
+        x0="-FourM_OuterWidth"/>
+      <envelope material="Steel235"/>
+
+      <eightmodule name="8MModule" vis= "LFHCAL8MModVis" repeat="0" >
+        <dimensions
+        width="EightM_OuterWidth"
+        height="EightM_OuterHeight"
+        widthBackInner="EightM_BackInnerWidth"
+        heightBackInner="EightM_BackInnerHeight"
+        widthSideWall="Mod_SideWallThickness"
+        widthTopWall="Mod_TopWallThickness"
+        thicknessMountingPlate="Mod_MountingPlateThickness"
+        thicknessFrontWall="Mod_FrontWallThickness"
+        thicknessBackWall="Mod_BackWallThickness"
+        arrayX="EightM_xArr"
+        arrayY="EightM_yArr"
+        notchDepth="Mod_NotchDepth"
+        notchHeight="Mod_NotchHeight"
+        foilThick="Mod_FoilThick"
+        pcbLength="Mod_PCBLength"
+        pcbThick="Mod_PCBThickness"
+        pcbWidth="Mod_PCBWidth"
+        pcbOffset="Mod_PCBOffset"
+        />
+        <position x="50*cm" y="50*cm" z="nHCal_LFHCAL_length - LFHCALElectronicsThickness" />
+      </eightmodule>
+      <fourmodule name="4MModule" vis= "LFHCAL4MModVis" repeat="2">
+        <dimensions
+        width="FourM_OuterWidth"
+        height="FourM_OuterHeight"
+        widthBackInner="FourM_BackInnerWidth"
+        heightBackInner="FourM_BackInnerHeight"
+        widthSideWall="Mod_SideWallThickness"
+        widthTopWall="Mod_TopWallThickness"
+        thicknessMountingPlate="Mod_MountingPlateThickness"
+        thicknessFrontWall="Mod_FrontWallThickness"
+        thicknessBackWall="Mod_BackWallThickness"
+        arrayX="FourM_xArr"
+        arrayY="FourM_yArr"
+        notchDepth="Mod_NotchDepth"
+        notchHeight="Mod_NotchHeight"
+        foilThick="Mod_FoilThick"
+        pcbLength="Mod_PCBLength"
+        pcbThick="Mod_PCBThickness"
+        pcbWidth="Mod_PCBWidth"
+        pcbOffset="Mod_PCBOffset"
+        />
+        <position x="50*cm" y="60*cm" z="nHCal_LFHCAL_length - LFHCALElectronicsThickness" />
+      </fourmodule>
+      <layer repeat="LFHCALLayer_NSteelRepeatSFirst" vis="InvisibleWithDaughters" readoutlayer="0">
```

Comment:
Are these constructed in order of decreasing $z$? Should we at least invert the indexing?